src/index.js

```diff
@@ -286,6 +285,7 @@ class WPTS {
 
         const options = seleniumAssistantBrowser.getSeleniumOptions();
         options.addArguments(`user-data-dir=${tempPreferenceFile}/`);
+        options.addArguments(`no-sandbox`);
```

Comment:
To be honest I do not fully understand what happens behind the scenes when this flag is on, but IIRC Chrome would not start up  when running in Docker on Ubuntu. I remember finding multiple mentions of this flag with regards to XVFB on Chrome mailing list but I did not keep the links.